volatility/framework/plugins/windows/dlllist.py

```diff
@@ -113,13 +127,27 @@ def _generator(self, procs):
                            proc.ImageFileName.cast("string",
                                                    max_length = proc.ImageFileName.vol.count,
                                                    errors = 'replace'), format_hints.Hex(entry.DllBase),
-                           format_hints.Hex(entry.SizeOfImage), BaseDllName, FullDllName, dumped))
+                           format_hints.Hex(entry.SizeOfImage), BaseDllName, FullDllName, DllLoadTime, dumped))
+
+    def generate_timeline(self):
+        for row in self._generator(pslist.PsList.list_processes(context = self.context,
+                                                                   layer_name = self.config['primary'],
+                                                                   symbol_table = self.config['nt_symbols'],
+                                                                   filter_func = pslist.PsList.create_pid_filter(None))):
```

Comment:
I hadn't considered generate_timeline being filterable.  It's probably fine, but I'll have a think.  For now this can stay as it is...

Reply:
(Given how timeliner runs, I don't think it's rigged to pass on parameters like pid to its children, so this may be a pointless line).